### ISSUE TITLE
Get trending songs by Region.

### ISSUE THREAD

**Issue by AnonymousXC** [sumitkolhe/jiosaavn-api#26]
Hi,
How can I get trending songs according to location. Like, localized song list.

**Comment by sumitkolhe** [comment]
i dont think you can specifically get trending songs for a particular location, you can get `What's tending in XYZ` location, but that includes artists, songs, playlists, albums, etc.

**Comment by Anuragd275** [comment]
Hey @AnonymousXC, JioSaavn currently does not support this feature for a local location, but for a Country/ Language/ Today.
